projects/vdk-plugins/vdk-impala/tests/conftest.py

```diff
@@ -16,18 +20,13 @@
 
 def _is_responsive(runner):
     try:
-        result = runner.invoke(["impala-query", "--query", "SELECT 1"])
-        if result.exit_code == 0:
-            return True
-    except:
-        return False
-
-
-@pytest.fixture(scope="session")
-def docker_compose_file(pytestconfig):
-    return os.path.join(
-        os.path.dirname(os.path.abspath(__file__)), "docker-compose.yml"
-    )
+        result: Result = runner.invoke(["impala-query", "--query", "SELECT 1"])
```

Comment:
do you really need the try catch here? 
Would be cleaner to just rmeove it and you aren't really adding much by wrapping it.

Reply:
Yes, because I need to re-wrap it in ConnectionError or the setup would fail (the wait_for waits only for ConnectionError and TimeOut error)

Reply:
I figured alternative way. I think it's more readable as well this way.